doc/cli_vs_yum.rst

```diff
@@ -99,6 +94,12 @@ following will work::
 
 ``include`` directive name of [main] and Repo configuration is a more logical and better named counterpart of ``exclude`` in DNF.
 
+=======================================================================
+``include`` option has been removed
```

Comment:
Please do use the definite particle (as @dmach suggested):
> **The** `include` option ...

Also, has the option been removed completely or only support for URLs has been removed? (below you still say "external configuration files")

Reply:
The `include` option was completely removed. I can clarify that. I also noticed that you self assigned the bug. Do you still want me to do the changes?

Reply:
I've assigned it to myself for review :slightly_smiling_face: (I do hope that's what it means...)

I just attempted to test this quickly and it does seem my `include=` was ignored.. I didn't find a deprecation warning in `/var/log/dnf.log` though... That's concerning :grin: (I do think we should have a deprecation warning for it - even if others may think differently - but don't worry about it here)